docs/pywbemclicommands.rst

```diff
@@ -535,42 +529,69 @@ The **instance** group defines commands that act on CIM instances including:
   WBEM server. The command build the CIMInstance from the class defined by
   CLASSNAME and the properties defined by the ``--property``\``-p`` option The
   properties are defined as name/value pairs, one property for each instance of
-  the ``--property`` option. Since the WBEM server (and pywbem) requires that
-  each property be typed, pywbemtools uses the CIMClass defined by CLASSNAME
-  retrieved from the WBEM server to define the type required to define the
-  CIMProperty.
+  the ``--property`` option.
 
-  For a single property in the new instance this is simply the `--property`` option
-  with the property name and value:
+  The ``--property`` argument value component defines the value of the
+  property. Since the WBEM server (and pywbem) requires that each property be
+  typed, pywbemcli uses the CIMClass defined by CLASSNAME retrieved from the
+  WBEM server to define the type required to define the CIMProperty. The value
+  argument contains just the value itself in the form of a string or numeric
```

Comment:
What is "The value argument"?
Suggest to change to "The option argument".

Reply:
DONE